tests/test_misc.py

```diff
@@ -229,41 +229,59 @@ def test_no_cosig_server(revault_network):
     rn.cancel_vault(vaults[-1])
 
 
+def get_unvault_txids(wallet, vaults):
+    unvault_txids = []
+    for vault in vaults:
+        deposit = f"{vault['txid']}:{vault['vout']}"
+        unvault_psbt = serializations.PSBT()
+        unvault_b64 = wallet.rpc.listpresignedtransactions([deposit])[
+            "presigned_transactions"
+        ][0]["unvault"]["psbt"]
+        unvault_psbt.deserialize(unvault_b64)
+        unvault_psbt.tx.calc_sha256()
+        unvault_txids.append(unvault_psbt.tx.hash)
+    return unvault_txids
+
+
 @pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
 def test_cpfp_transaction(revault_network, bitcoind):
     CSV = 12
     revault_network.deploy(
         2,
         1,
         csv=CSV,
-        with_watchtowers=False,
+        # with_watchtowers=False,
```

Comment:
Don't comment, either you set to true or to false

Reply:
I don't recall any real reason, probably I followed a "if watchtowers are not needed here why spawning them at all" kinda reasoning to make the test run quicker

Reply:
Makes sense. My own rationale for making it default to `True` was: "test in as many scenarii as possible".